tests/test_iib_ops.py

```diff
@@ -293,10 +343,20 @@ def remove_operators_mock_calls_tester(
     "extra_args,push_items,mock_calls_tester",
     [
         (
-            [],
+            ["--deprecation-list", "bundle1,bundle2"],
+            [operator_1_push_item_pending, operator_1_push_item_pushed],
+            add_bundles_mock_calls_tester_deprecation_bundles,
+        ),
+        (
+            ["--deprecation-list", "bundle1"],
             [operator_1_push_item_pending, operator_1_push_item_pushed],
             add_bundles_mock_calls_tester,
         ),
+        (
```

Comment:
Can I ask what happens if `--deprecation-list` is `None`

Reply:
Line #538 in iib_ops.py. It will not take an argument.

Reply:
As in the output I pasted above, I don't think #538 will work. `None` would essentially mean not even an empty string and simply the argument. So it'll falter even before it reaches #538